src/app/comments/comment-form-container/comment-form/comment-form.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {AbstractControl, FormBuilder, FormGroup, Validators} from "@angular/forms";
+
+
+
+@Component({
+  selector: 'app-comment-form',
+  templateUrl: './comment-form.component.html',
+  styleUrls: ['./comment-form.component.css']
+})
+export class CommentFormComponent implements OnInit {
+
+  @Input()
+  successMessage
+
+  @Input()
+  errorMessage
+
+  commentForm: FormGroup = this.fb.group({
+    firstName: [null,
+        [
+          Validators.required,
+          Validators.minLength(5),
```

Comment:
good practice to keep all magic numbers in variables, here in component as static class variables, eg. MIN_INPUT_LENGTH = 5 and so on.